python/pyspark/heapq3.py

```diff
@@ -883,7 +883,7 @@ def nlargest(n, iterable, key=None):
 
 
 if __name__ == "__main__":
-    import doctest
+    import doctest, sys
```

Comment:
Shell we import this separately? - https://www.python.org/dev/peps/pep-0008/#imports